website/static/js/registrationUtils.js

```diff
@@ -599,11 +609,26 @@ RegistrationEditor.prototype.check = function() {
 
     var proceed = true;
     $.each(self.flatQuestions(), function(i, question) {
-        var valid = question.valid();
-        proceed = proceed && valid.status;
+        if ( question.required ) {
+            var valid = question.value.isValid();
+            proceed = proceed && valid.status;
+        }
     });
     if (!proceed) {
-        self.showValidation(true);
+
+        bootbox.dialog({
+            title: "Registration not complete",
+            message: "There are errors in your registration. Please double check it and submit again",
```

Comment:
I think the capitalization is off—I think the title should be entirely capitalized (check with Lauren). And I'd add a period to the last sentence. Otherwise I think the language is fine.

If it's possible to give more information on why it failed though, that's always helpful.